x-pack/platform/packages/private/ml/url_state/src/url_state.test.tsx

```diff
@@ -143,3 +149,89 @@ describe('useUrlState', () => {
     expect(getByTestId('appState').innerHTML).toBe('the updated query');
   });
 });
+
+describe('usePageUrlState', () => {
```

Comment:
what is the motivation of using components in tests? did yo try rendering the hook itself?

Reply:
good point, I just aligned with the existing tests, but rendering the test component seems redundant, updated in: [#7df5acb](https://github.com/elastic/kibana/pull/203224/commits/7df5acb257026f016ab3f8744ed6e4b0173b8278)